src/main/java/com/lpvs/service/LPVSDetectService.java

```diff
@@ -68,17 +76,31 @@ private void init() {
 
     @EventListener(ApplicationReadyEvent.class)
     public void runOneScan() {
+        log.info("Triggered signle scan operation");
         if (trigger != null && !HtmlUtils.htmlEscape(trigger).equals("")) {
             try {
                 LPVSQueue webhookConfig =
                         this.getInternalQueueByPullRequest(HtmlUtils.htmlEscape(trigger));
-                this.runScan(webhookConfig, LPVSDetectService.getPathByPullRequest(webhookConfig));
-                File scanResult = new File(LPVSFileUtil.getScanResultsJsonFilePath(webhookConfig));
-                if (scanResult.exists()) {
-                    String jsonTxt = IOUtils.readFileToString(scanResult);
-                    // ToDo: form html report and console output
-                    log.info(jsonTxt);
-                    log.info("\n\n\n Single scan finished successfully \n\n\n");
+
+                scanossDetectService.runScan(
+                        webhookConfig, LPVSDetectService.getPathByPullRequest(webhookConfig));
+                List<LPVSFile> scanResult = scanossDetectService.checkLicenses(webhookConfig);
+
+                List<LPVSLicenseService.Conflict<String, String>> detectedConflicts =
+                        licenseService.findConflicts(webhookConfig, scanResult);
+
+                if (buildReport != null
+                        && !HtmlUtils.htmlEscape(buildReport).equals("")
+                        && Files.exists(Paths.get(buildReport))) {
+                    String report =
+                            LPVSCommentUtil.buildHTMLComment(
+                                    webhookConfig, scanResult, detectedConflicts);
+                    LPVSCommentUtil.saveHTMLToFile(report, buildReport + "/LPVSreport.html");
+                } else {
+                    String report =
+                            LPVSCommentUtil.reportCommentBuilder(
+                                    webhookConfig, scanResult, detectedConflicts);
+                    log.info(report);
                 }
             } catch (Exception ex) {
                 log.info("\n\n\n Single scan finished with errors \n\n\n");
```

Comment:
Maybe `log.error` is a better choise?

Reply:
next message is error with reporting. But we may change this as well.